base/experimental.jl

```diff
@@ -375,4 +375,18 @@ adding them to the global method table.
 """
 :@MethodTable
 
+struct GetPropertyLens
+    syms::Tuple{Vararg{Symbol}}
+end
```

Comment:
Couldn't we use `Fix1{getproperty}` for this instead of defining a new type, if `getproperty` is extended to take a tuple of symbols as the second argument?

Reply:
Yes, we can. The original idea here was to have something more first class, but you can always turn `Fix1{getproperty}` into that by evaluating it on an appropriate tracing object. Since that's a more minimal change, I'll use that instead.

Reply:
I am somewhat surprised we can const-prop that reliably enough (esp to something like broadcast which tries not to inline this much) without putting the symbols in the type parameters of the Lens object. Should we put it there instead?